File: frontend/lib/domain/entities/student.dart
class Student {
  const Student({
    required this.id,
    required this.name,
    this.registryCode,
    this.email,
    this.phone,
    this.notes,
  });

  final String id;
  final String name;
  final String? registryCode;
  final String? email;
  final String? phone;
  final String? notes;
}

File: backend/src/modules/students/domain/student.ts
export type Student = {
  id: string;
  teacherId: string;
  name: string;
  registryCode: string | null;
  email: string | null;
  phone: string | null;
  notes: string | null;
  createdAt: Date;
  updatedAt: Date;
  deletedAt: Date | null;
};
